### PR TITLE
PAE-602 - Responsive login pearson-vue-theme

### PR DESCRIPTION
**Pull request by luismorenolopera**
Description
----------------
different screen sizes are supported for pearson-vue-theme.

Screenshots
-----------------

**360x640:**

image

**411x731:**

image

**1024x768:**

image

**540x720:**

image

**1980x1080:**

image

**Ticket:** [PAE-602](https://pearsonadvance.atlassian.net/browse/PAE-602)

### CODE REVIEW

File: edx-platform/pearson-vue-theme/lms/static/sass/partials/lms/theme/_extras.scss [hunk 1]
```diff
@@ -380,22 +368,162 @@ label[for=register-password]{
...
+  }
+
+  .background-login {
+    display: none;
+  }
+
+  .signin-container {
+    top: 260px;
+    width: 230px;
+    box-shadow: 2px 1px 1px 3px #bcb9b9;
```
Comment (Squirrel18) [inline edx-platform/pearson-vue-theme/lms/static/sass/partials/lms/theme/_extras.scss]: Please add a variable for this colour definition. @luismorenolopera

File: edx-platform/pearson-vue-theme/lms/static/sass/partials/lms/theme/_extras.scss [hunk 2]
```diff
@@ -380,22 +368,162 @@ label[for=register-password]{
...
+  }
+
+  .login-register div.toggle-form {
+
```
Comment (Squirrel18) [inline edx-platform/pearson-vue-theme/lms/static/sass/partials/lms/theme/_extras.scss]: Unnecessary blank line. @luismorenolopera

File: edx-platform/pearson-vue-theme/lms/static/sass/partials/lms/theme/_extras.scss [hunk 3]
```diff
@@ -380,22 +368,162 @@ label[for=register-password]{
...
+  .background-login {
+    display: none;
+  }
+
+  .signin-container {
+    top: 280px;
+  }
+
+  .login-register div.toggle-form {
+
```
Comment (Squirrel18) [inline edx-platform/pearson-vue-theme/lms/static/sass/partials/lms/theme/_extras.scss]: Unnecessary blank line. @luismorenolopera

File: edx-platform/pearson-vue-theme/lms/templates/student_account/login.underscore [hunk 4]
```diff
@@ -55,3 +55,9 @@
     </form>
   </div>
 </div>
+<style media="screen">
+
+  @media (min-width: 1280px) {
+
+  }
+</style>
```
Comment (Squirrel18) [inline edx-platform/pearson-vue-theme/lms/templates/student_account/login.underscore]: Remove this block. @luismorenolopera

**Review by Squirrel18 — CHANGES_REQUESTED** [review]

File: edx-platform/pearson-vue-theme/lms/static/sass/partials/lms/theme/_extras.scss [hunk 5]
```diff
@@ -323,9 +308,12 @@ iframe.rocket-chat-view {
   display: block;
 }
 
-.login-container {
-   position: absolute;
-   right: 450px;
```
Comment (ivanvgh) [inline edx-platform/pearson-vue-theme/lms/static/sass/partials/lms/theme/_extras.scss:328]: @luismorenolopera, Can we keep the same indentation throughout the file?

**Review by Squirrel18 — APPROVED** [review]

File: edx-platform/pearson-vue-theme/lms/static/sass/partials/lms/theme/_extras.scss (hunk 5 above)
Comment (luismorenolopera) [reply edx-platform/pearson-vue-theme/lms/static/sass/partials/lms/theme/_extras.scss:328]: done

**Review by ivanvgh — APPROVED** [review]

**Pull request merged**